Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -2814,6 +2818,9 @@ class CloneDeserializer : CloneBase {
         case CryptoAlgorithmIdentifierTag::ECDH:
             result = CryptoAlgorithmIdentifier::ECDH;
             break;
+        case CryptoAlgorithmIdentifierTag::Ed25519:
+            result  = CryptoAlgorithmIdentifier::Ed25519;
+            break;
```

Comment:
It might be good to add a test that validates Ed25519 key serialisation works.

Reply:
What do you mean about this?

Reply:
done